cvs -z9 diff treelay.h (in directory E:\Dev\wxWindows\include\wx\generic\)
Index: treelay.h
===================================================================
RCS file: /home/wxcvs/wxWindows/include/wx/generic/treelay.h,v
retrieving revision 1.5
diff -r1.5 treelay.h
20,23c20,24
< #include <wx/object.h>
< class wxList;
< class wxDC;
< class wxMouseEvent;
---
> #include "wx/string.h"
> #include "wx/object.h"
> #include "wx/list.h"
> #include "wx/dc.h"
> #include "wx/event.h"
25,26d25
< 
< #include <wx/string.h>


 	  	 
